dist/test_tag.py

```diff
@@ -0,0 +1,143 @@
+import os
+
+##### LOCAL VARIABLES AND CONSTANTS #####
+component = ""
+testingArea = ""
+testType = ""
+
+isParsingTag = False
+
+nbTests = 0
+
+sortedTags = []
+testFiles = []
+
+taggedFiles = {}
+validTags = {}
+
+END_TAG = "[END_TAGS]"
+NB_TAG_ARGS = 3
+START_TAG = "[TEST_TAGS]"
+#####
+
+##### GET ALL TEST FILES #####
+for root, dirs, files in os.walk("../test/"):
+    path = root.split(os.sep)
+    for file in files:
+        filename = os.path.join('/'.join(path), file)
+        if filename.endswith("main.c") or filename.endswith(".py"):
+            testFiles.append(filename)
+#####
+
+##### RETRIEVE VALID TAGS #####
+validationFile = open("test_tags.ok", "r")
+
+# The file has the following pattern
+# <COMPONENT>:<TESTING_TYPE>:<TESTING_AREA>:<DESCRIPTION>
+# A tag is made of the three first values: COMPONENT, TEST_TYPE and TESTING_AREA
+tags = validationFile.readlines()
+tags = [tag.replace('\n', '') for tag in tags]
+
+for tag in tags:
+    currentLine = tag.split(':')
+    # Createa key value pair <TAG>:<DESCRIPTION>
+    validTags[':'.join(currentLine[:NB_TAG_ARGS])] = ':'.join(currentLine[NB_TAG_ARGS:])
+
+validationFile.close()
+#####
+
+##### PARSE TEST FILES #####
+for filename in testFiles:
+    inputFile = open(filename, "r")
+
```

Comment:
There's a lot of unnecessary white space in this file can we remove some of it. e.g. this line, line 57, 69, 82,91.